### ISSUE TITLE
python 3.7 issues

### ISSUE THREAD

**Issue by aerickson** [amey-sam/Flask-MailGun#21]
```
File REDACTED/venv/src/flask-mailgun3/flask_mailgun/processing.py", line 30
    self.async = async_pool(app.config.get('MAILGUN_BG_PROCESSES', 4))
             ^
SyntaxError: invalid syntax
```

Seems related to python 3.7 changes. See https://github.com/ansible/ansible/issues/32816.

**Comment by pahrohfit** [comment]
pull request to fix this submitted and passed tests.

**Comment by Richard-Mathie** [comment]
fixed by #25 #24 #23
